build.gradle

```diff
@@ -135,6 +135,8 @@ sourceSets {
         proto {
             srcDir 'src/main/proto'
             exclude "apigrpc.proto"
+            exclude "github.com/heroiclabs/nakama-common/api/*.proto"
```

Comment:
I thought `apigrpc.proto` exclusion would be enough to not include the remainder of the `api` in Satori, but I have to explicitly exclude that too.